### PR TITLE
Ingress removal from IM

### PR DESCRIPTION
**Pull request by ujjwalchk-it**
Remove ngix and ingress dependency.
Create the below artefacts from im-operator which was managed by ingress operator -

- ibmcloud-cluster-info configmap
- ibmcloud-cluster-ca-cert secret with ca-cert

Ref https://github.ibm.com/IBMPrivateCloud/roadmap/blob/master/feature-specs/bedrock-services-1Q2023/Certificate%2C%20Route%2C%20Shared%20Secret%20and%20Configmap%20management%20in%20Cloud%20Pak%203.0.md

### CODE REVIEW

File: pkg/controller/authentication/secret.go [hunk 1]
```diff
@@ -150,3 +171,97 @@ func generateSecretObject(instance *operatorv1alpha1.Authentication, scheme *run
...
+func (r *ReconcileAuthentication) createClusterCACert(i *operatorv1alpha1.Authentication, scheme *runtime.Scheme, secretName, ns string, caCert []byte) error {
+
+	reqLogger := log.WithValues("Instance.Namespace", i.Namespace, "Instance.Name", i.Name)
+
+	// create ibmcloud-cluster-ca-cert
+	labels := map[string]string{
+		"app":                          "auth-idp",
+		"component":                    "auth-idp",
+		"app.kubernetes.io/component":  "auth-idp",
+		"app.kubernetes.io/name":       "auth-idp",
```
Comment (rashmi43) [inline pkg/controller/authentication/secret.go:202]: no auth-idp deployment anymore, we may have to update this label

**Review by rashmi43 — APPROVED** [review]
there is not auth-idp

**Review by Tirumalavasa — APPROVED** [review]
/lgtm

**Pull request merged**